airflow/templates/configmaps.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: "{{ .Values.airflow.prefix -}}airflow-cfg"
+  name: {{ template "airflow.fullname" . }}-airflow-cfg
```

Comment:
I don't exactly see the point?

Don't you want something like
```
{{ template "airflow.fullname" airflow-cfg }}
```

Reply:
In the docs they have something like this

apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ .Release.Name }}-configmap

I was mostly trying  to conform to this. I haven't tried this syntax but I can.

Reply:
ok i'll see what this do tomorow. My requirement is to be able to change this release easily so we can have several instance of airflow in the same namespace (especially, we share the namespace with another celery+rabbitmq), but we are also trying kubernetes executor fork, all in the same namespace